<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>phase</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>hkl</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>d-space</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="d_space_peak">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>250</horstretch>
       <verstretch>150</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="d_space_type">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Select Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fourier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_quadratic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_closed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_open</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>independent</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="dspace_checkBox">
     <property name="text">
      <string>fixed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="dspace_fixed">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="height_peak">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>250</horstretch>
       <verstretch>150</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="height_peak_type">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Select Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fourier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_quadratic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_closed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_open</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>independent</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="height_checkBox">
     <property name="text">
      <string>fixed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="height_fixed">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="profile_peak">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>250</horstretch>
       <verstretch>150</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="profile_peak_type">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Select Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fourier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_quadratic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_closed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_open</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>independent</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="profile_checkBox">
     <property name="text">
      <string>fixed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="profile_fixed">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>width</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="width_peak">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>250</horstretch>
       <verstretch>150</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="width_peak_type">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Select Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fourier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_linear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_quadratic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_closed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline_cubic_open</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>independent</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="width_checkBox">
     <property name="text">
      <string>fixed</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="width_fixed">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>symmetry</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="phase_peak">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="hkl">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QLineEdit" name="symmetry_peak">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
